google-cloud-clients/google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITDMLTest.java

```diff
@@ -82,6 +86,11 @@ private void executeUpdate(long expectedCount, final String... stmts) {
           public Long run(TransactionContext transaction) {
             long rowCount = 0;
             for (String stmt : stmts) {
+              if (throwAbortOnce) {
+                throwAbortOnce = false;
+                throw SpannerExceptionFactory.newSpannerException(ErrorCode.ABORTED, "Abort in test");
```

Comment:
From the code formatter:
```suggestion
                throw SpannerExceptionFactory.newSpannerException(
                    ErrorCode.ABORTED, "Abort in test");
```